libraries/EmbeddedChessPieces/src/ECPMovement/ECPMovement.h

```diff
@@ -0,0 +1,68 @@
+/**
+ * @file ECPMovement.h
+ * @author Ines Rohrbach, Nico Schramm
+ * @brief Helper class for chess piece movement.
+ * @version 0.1
+ * @date 2024-11-12
+ * 
+ * @copyright Copyright (c) 2024
+ * 
+ */
+
+#ifndef ECPMovement_h
+#define ECPMovement_h
+
+#include <Dezibot.h>
+#include <EmbeddedChessPieces.h>
+
+#define FORWARD_TIME 750
+#define MOVEMENT_BREAK 375
+
+
+class ECPMovement {
```

Comment:
```suggestion
#define MOVEMENT_BREAK 375

class ECPMovement {
```

Please only leave one new line here to keep it consistent throughout the project \:)